amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -220,10 +232,19 @@ vx_status SequenceRearrange_Register(vx_context);
 #define VX_KERNEL_RPP_COPY_NAME                                              "org.rpp.Copy"
 #define VX_KERNEL_RPP_BRIGHTNESS_NAME                                        "org.rpp.Brightness"
 #define VX_KERNEL_RPP_GAMMACORRECTION_NAME                                   "org.rpp.GammaCorrection"
+#define VX_KERNEL_RPP_BLEND_NAME                                             "org.rpp.Blend"
+#define VX_KERNEL_RPP_EXPOSURE_NAME                                          "org.rpp.Exposure"
+#define VX_KERNEL_RPP_WARPAFFINE_NAME                                          "org.rpp.WarpAffine"
+#define VX_KERNEL_RPP_ROTATE_NAME                                             "org.rpp.Rotate"
+
 #define VX_KERNEL_RPP_CROPMIRRORNORMALIZE_NAME                               "org.rpp.CropMirrorNormalize"
 #define VX_KERNEL_RPP_RESIZE_NAME                                            "org.rpp.Resize"
 #define VX_KERNEL_RPP_CROP_NAME                                              "org.rpp.Crop"
 #define VX_KERNEL_RPP_COLORTWIST_NAME                                        "org.rpp.ColorTwist"
 #define VX_KERNEL_RPP_RESIZEMIRROR_NAME                                      "org.rpp.ResizeMirrorNormalize"
 #define VX_KERNEL_RPP_SEQUENCEREARRANGE_NAME                                 "org.rpp.SequenceRearrange"
+#define VX_KERNEL_RPP_CONTRAST_NAME                                          "org.rpp.Contrast"
+#define VX_KERNEL_RPP_FLIP_NAME                                              "org.rpp.Flip"
```

Comment:
Either add everything to the end or the beginning please

Reply:
Resolved